libs/blocks/accordion/accordion.js

```diff
@@ -10,7 +11,39 @@ function setSEO(questions) {
   document.head.append(script);
 }
 
-function handleClick(el, dd) {
+function displayMedia(displayArea, el, dd, i, id, expanded) {
+  const theID = el.getAttribute('aria-controls').split('-')[1];
+  [...mediaCollection[theID]].map(
+    (mediaCollectionItem, idx, total) => {
+      mediaCollectionItem.classList.remove('expanded');
+
+      for (let index = 0; index < total.length; index++) {
+        const trigger = document.querySelector(`#accordion-${theID}-trigger-${index + 1}`);
+        const content = document.querySelector(`#accordion-${theID}-content-${index + 1}`);
+        trigger.setAttribute('hidden', '');
+        trigger.setAttribute('aria-expanded', 'false');  
+        content.setAttribute('hidden', '');
+        content.setAttribute('aria-expanded', 'false');
+      }
+      el.setAttribute('aria-expanded', 'true');
```

Comment:
can you move this into an if/else with `if (expanded)` so that attributes are not set twice